### PR TITLE
Make Dashboard Sessions Non-Sticky

### PR DESCRIPTION
**Pull request by abh1kg**
- Implement persistent sessions with support for Mongo + Couch
- Allow pure scalability for dashboard

### CODE REVIEW

File: lib/cf/dashboard/manifest.yml [hunk 1]
```diff
@@ -5,10 +5,17 @@ applications:
...
+   AUTH_SERVER: https://uaa
+   CF_CLIENT_ID: abacus_db_client
+   CF_CLIENT_SECRET: secret
+   CF_COOKIE_SECRET: secret
+   NODE_TLS_REJECT_UNAUTHORIZED: 0
+   PROVISIONING: abacus-provisioning-plugin
+   DEBUG: abacus-*
+   DBALIAS: db-session
+   DBCLIENT: abacus-mongoclient
+   AUTOREMOVEINTERVAL: 10
```
Comment (hsiliev) [inline lib/cf/dashboard/manifest.yml]: Can we have this in with underscores: `AUTO_REMOVE_INTERVAL`, as the rest of the variables?

File: lib/cf/dashboard/manifest.yml [hunk 2]
```diff
@@ -5,10 +5,17 @@ applications:
...
+  services:
+  - db-session
   env:
-    API: https://api
-    AUTH_SERVER: https://uaa
-    CF_CLIENT_ID: abacus_db_client
-    CF_CLIENT_SECRET: secret
-    CF_COOKIE_SECRET: secret
-    PROVISIONING: abacus-provisioning-plugin
+   API: https://api
```
Comment (hsiliev) [inline lib/cf/dashboard/manifest.yml]: Don't we need 2 spaces offset?

File: lib/cf/dashboard/package.json [hunk 3]
```diff
@@ -42,13 +42,17 @@
 		"coverage": "karma start src/test/frontend/karma.conf.js && node ./node_modules/istanbul-harmony/lib/cli.js cover  --hook-run-in-context ./node_modules/mocha/bin/_mocha -- --ui bdd -R spec -t 5000 src/test/backend/ && node --harmony ./node_modules/istanbul-harmony/lib/cli.js report --dir src/test/backend/coverage --root coverage html && node --harmony src/test/computeCoverage.js"
 	},
 	"dependencies": {
+		"abacus-vcapenv": "file:../../../abacus/lib/utils/vcapenv",
+		"abacus-dbclient": "file:../../../abacus/lib/utils/dbclient",
 		"abacus-cluster": "file:../../../abacus/lib/utils/cluster",
+		"abacus-couchstore": "file:../../../abacus/lib/utils/couchstore",
```
Comment (hsiliev) [inline lib/cf/dashboard/package.json]: You also need to bump Abacus submodule to a commit that contains the couchstore

**Comment by hsiliev** [comment]
Change looks good, but there are conflicts. @abh1kg Can you please resolve them?

**Comment by abh1kg** [comment]
@hsiliev : The merge conflicts are resolved now

**Pull request merged**